07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,73 @@
+#header {
```

Comment:
Great job! Try centering it on the page with some margin auto and width: XX% on the body.